docs/getting-started/pingfederate.md

```diff
@@ -0,0 +1,158 @@
+# Getting Started - PingFederate
+
+<div class="banner" onclick="window.open('https://registry.terraform.io/providers/pingidentity/pingfederate/latest','');">
+    <img class="assets" src="../../img/logos/tf-logo.svg" alt="Terraform logo" />
+    <span class="caption">
+        <a class="assetlinks" href="https://registry.terraform.io/providers/pingidentity/pingfederate/latest" target=”_blank”>Registry</a>
+    </span>
+</div>
+
+## EARLY ACCESS
+
+As of January 2024, the PingFederate Terraform provider is an early access release. **This status indicates that the provider is in active development and is subject to breaking changes. While available in the Terraform Registry, use of this provider is not recommended for production use at this time.**
+
+## Requirements
+
+* Terraform CLI 1.4+
+* A running PingFederate server accessible over HTTPS, or Docker CLI to start one
+* If using Docker to start a PingFederate server, a DevOps license is required - [Register for the DevOps program here](https://devops.pingidentity.com/how-to/devopsRegistration/)
+
+## Start a PingFederate Docker image to be configured
+
+!!! warning "Using an Existing PingFederate Server"
+    If you already have a running PingFederate server that you can reach over HTTPS, you can skip this step. The provider can be used with any PingFederate server.
+
+First, start a PingFederate server using Docker. Your DevOps credentials will be read from the `${HOME}/.pingidentity/config` file. The HTTPS port (default `9999`) must be exposed.
+
+```shell
+docker run --name pingfederate_terraform_provider_container \
+  -d -p 9031:9031 \
+  -d -p 9999:9999 \
+  --env-file "${HOME}/.pingidentity/config" \
+  -e SERVER_PROFILE_URL=https://github.com/pingidentity/pingidentity-server-profiles.git \
+  -e SERVER_PROFILE_BRANCH=terraform-provider-pingfederate-1125 \
+  -e SERVER_PROFILE_PATH=terraform-provider-pingfederate/pingfederate \
+  pingidentity/pingfederate:$${PINGFEDERATE_PROVIDER_PRODUCT_VERSION:-12.0.0}-latest
+```
+
+After starting the container, follow the logs until the server becomes available.
+
+```shell
+docker logs -f pingfederate_terraform_provider_container
+```
+
+After you see the following message in the container logs, the server is ready to receive requests from the provider:
+
+```shell
+PingFederate is up
+```
+
+## Accessing the PingFederate API
+
+Using the port mappings from the example above, PingFederate will be available at `https://localhost:9999/pingfederate/app#/`.
```

Comment:
should we follow something more like PF docs:
"On your OIDC provider, configure an OAuth client to represent the PingFederate administrative console. Specify the following redirect URI for the client:
https://<pf_admin_hostname>:<pf_admin_port>/pingfederate/app?service=finishsso
or, if using pf.admin.baseurl:
https://<pf.admin.baseurl>/pingfederate/app?service=finishsso "
from: https://docs.pingidentity.com/r/en-us/pingfederate-110/pf_enabling_oidc_based_auth
or:
"Browse to the following URL: https://<pf_host>:9999/pf-admin-api/api-docs/
Note:

<pf_host> is the network address of your PingFederate server. It can be an IP address, a host name, or a fully qualified domain name. It must be reachable from your computer.

9999 is the default value of the pf.admin.https.port property in the run.properties file."
from: https://docs.pingidentity.com/r/en-us/pingfederate-112/pf_access_api_interact_documentation

Reply:
we could make the clarification earlier in the doc